DscResourceTestHelper/TestHelper.psm1

```diff
@@ -319,7 +319,7 @@ function Initialize-TestEnvironment
     } 
     
     # The folder where this module is found
-    [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
+    [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path)))
```

Comment:
isn't it going too deep?
e.g. 
xWebAdministration\Tests\DscResourceTestHelper\TestHelper.psm1
this way we need 3 Split-Path -Parent to get the root